arm-network/2015-06-15/swagger/network.json

```diff
@@ -6129,9 +6128,6 @@
           "description": "Gets or sets Provisioning state of the PublicIP resource Updating/Deleting/Failed"
         }
       },
-      "required": [
-        "publicIPAllocationMethod"
```

Comment:
Commenting it here, since GitHub does not allow me to comment on unchanged parts. I wanted to verify that, do you really want to model publicIPAllocationMethod as a string (as specified in the extension `"x-ms-enum" : {"modelAsString : true }`)? Because by modeling it as string you don't get any validation. Please read the docs over [here.](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#Enum-x-ms-enum)